test/ExternalBribes.t.sol

```diff
@@ -27,7 +27,7 @@ contract ExternalBribesTest is BaseTest {
         mintFlow(owners, amounts);
         mintLR(owners, amounts);
         VeArtProxy artProxy = new VeArtProxy();
-        escrow = new VotingEscrow(address(VELO), address(artProxy));
+        escrow = new VotingEscrow(address(FLOW), address(artProxy), owners[0]);
```

Comment:
An array of address that can update the art?

Reply:
nft marketplace will identify collection owner with `owner`.